diff --git a/node_modules/react-draggable/typings/index.d.ts b/node_modules/react-draggable/typings/index.d.ts
index ad7f22b..31be3d9 100644
--- a/node_modules/react-draggable/typings/index.d.ts
+++ b/node_modules/react-draggable/typings/index.d.ts
@@ -54,6 +54,7 @@ declare module 'react-draggable' {
     onStop: DraggableEventHandler,
     onMouseDown: (e: MouseEvent) => void,
     scale: number
+    children?: ReactNode
   }
 
   export default class Draggable extends React.Component<Partial<DraggableProps>, {}> {
